src/cli_plugin/install/download.js

```diff
@@ -29,7 +32,8 @@ export function download(settings, logger) {
 
     return _downloadSingle(settings, logger, sourceUrl)
     .catch((err) => {
-      if (err.message === 'ENOTFOUND') {
+      if ((err instanceof UnsupportedProtocolError)
```

Comment:
No need to wrap each individual condition in parenthesis.

We also tend to start breaking these things down into descriptive variables to when we think the conditional is pretty complex.  So something like:

```
const isUnsupported = err instanceof UnsupportedProtocolError;
const isNotFound = err.message === 'ENOTFOUND';
if (isUnsupported && isNotFound) {
  return tryNext();
}
```